indico/modules/events/templates/display/indico/_common.html

```diff
@@ -89,6 +89,7 @@
 
 {% macro render_datetime(prop_name, dt, timezone, format='medium') -%}
     <time itemprop="{{ prop_name }}" datetime="{{ dt.isoformat() }}">
+        {{ dt.strftime('%A') }}
```

Comment:
trans is used to translate static text; but if you use `format_date` it'll properly translate weekdays etc